tests/app/ios/BUILD.bazel

```diff
@@ -0,0 +1,16 @@
+load("//rules:framework.bzl", "apple_framework")
+load("//rules:app.bzl", "ios_application")
+
+apple_framework(
+    name = "FW",
+    srcs = glob(["FW/*"]),
+    private_headers = glob(["FW/*_Private.*"]),
+)
+
+ios_application(
```

Comment:
This is just verifying that we can build frameworks generally, but not necessarily that the clean action is working, correct? Just want to make sure I'm understanding correctly the intent.

Reply:
Correct -- I needed a testbed to make changes in, and there wasn't one with apps and frameworks, so I added it